pontoon/administration/files.py

```diff
@@ -770,9 +768,14 @@ def update_from_repository(project, locales=None):
         repository_url_master = repository_url.rsplit(ending, 1)[0]
         repository_path = os.path.join(repository_path_master, ending)
 
+    # If Mozilla repo, set paths
```

Comment:
Set paths to what? If the repo URL points to your special checkouts then we set the URL we're pulling updates from to be... the official mozilla HG repo? I thought we were supposed to be updating from your special version instead?